frameworks/OCaml/webmachine/src/src/bin/tfb.ml

```diff
@@ -138,7 +139,46 @@ class queries =
       Wm.continue (`String (Lib.Db_j.string_of_queries json)) rd
   end
 
+let dump_lwt () =
+  let options =
+    [
+      ("fd_passing", `fd_passing);
+      ("fdatasync", `fdatasync);
+      ("get_affinity", `get_affinity);
+      ("get_cpu", `get_cpu);
+      ("get_credentials", `get_credentials);
+      ("libev", `libev);
+      ("madvise", `madvise);
+      ("mincore", `mincore);
+      ("recv_msg", `recv_msg);
+      ("send_msg", `send_msg);
+      ("set_affinity", `set_affinity);
+      ("wait4", `wait4);
+    ]
+  in
+  ignore @@ Lwt_io.eprintf "Lwt:\n";
+  List.iter
+    (fun (str, opt) ->
+      ignore @@ Lwt_io.eprintf "  %s = %b\n" str (Lwt_sys.have opt))
+    options
+
 let main () =
+  (* https://github.com/mirage/ocaml-cohttp/issues/328#issuecomment-222583580 *)
+  Lwt_io.set_default_buffer_size 0x10000;
+  let nproc =
+    match Sys.getenv "CORE_COUNT" with
```

Comment:
why not use all cores?

Reply:
I wanted to compare the forking shared socket approach with a multiprocess multisocket approach while still using your tweaks to memoize the `Date` header.